src/pages/components/table-of-contents.mdx

```diff
@@ -42,14 +51,26 @@ The area above the Table of contents can be utilized in many ways—showing navi
 <Row>
 <Column colMd={8} colLg={8}>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
+![Image of With heading content variation](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
 
 </Column>
 </Row>
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
+![Image of With heading content variation usecase](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
```

Comment:
Are we sharing with users about the escape hatch above the table of contents? Especially the moment about images? Also, it's only in the vertical scenario I believe, so that should be under vertical from my point of view from a hierarchy standpoint.

Reply:
Are you suggesting taking the "With heading content" off the page? @photodow

Reply:
Well either take it off or move it under the vertical/default guidance. I thought we decided we didn't want to advertise it, but if I misunderstood that then that's cool too. Hierarchy is off at the very least I think.

Reply:
Agreed it is an addition to the Default/vertical. @JennySanchez This was part of why I wondered if Vertical and Horizontal should have their own anchor links like we do with Masthead L0 and Masthead L1, to establish the correct hierarchy. I may have not explained that very well when we met. 

In storybook we are planning to move the 'with heading content' into the Vertical display:
<img width="372" alt="Screen Shot 2021-03-26 at 4 04 47 PM" src="https://user-images.githubusercontent.com/45692622/113455779-82686c80-93d9-11eb-8595-3c2ab6c1c7b9.png">